src/pages/LoginView.jsx

```diff
@@ -28,74 +24,89 @@ function LoginView() {
    * Helper Functions
    ************************************/
 
- /* Handler for log in button clicked, if valid inputs logs in user
-  */
+  /* Handler for log in button clicked, if valid inputs logs in user
+   */
   async function loginClicked() {
     if (validateInput()) {
       const response = await loginUser(email, password);
 
       if (response === SUCCESS) {
         setIsLoggedIn(true);
-      }
-      else {
+      } else {
         setErrorMessage(response);
-      } 
+      }
     }
   }
 
- /* Validates inputs, if invalid then it will display an error message
-  */
+  /* Validates inputs, if invalid then it will display an error message
+   */
   function validateInput() {
-    const emailResponse = validateEmail(email);
-    const passwordResponse = validatePassword(password);
+    handleEmailValidationResponse(email);
+    handlePasswordValidationResponse(password);
 
-    if (emailResponse !== SUCCESS) {
-      setEmailErrorMessage(emailResponse);
-    }
-    else {
-      setEmailErrorMessage(null);
-    }
+    return email !== '' && password !== '';
```

Comment:
Is there an edge case for empty string checking?

Reply:
this was done since we wanted to use the backend to validate email and password for Login. only need to check for empty string